pages/doc/proxies_versions.md

```diff
@@ -14,13 +14,15 @@ This page gives an overview of important changes for the most recent Wavefront p
 ## Version 6.1
 
 - Significantly improved data parsing performance and throughput.
+- Redesigned and improved storage engine for spooling data to disk.
+  {% include note.html content="The new proxy stores spooled data differently from previous versions, so if your proxies have accumulated substantial backlog, we recommend draining the backlog first before upgrading to version 6.1 or newer." %}
```

Comment:
The new proxy stores spooled data differently from previous versions, so if your proxies have accumulated substantial backlog, we recommend draining the backlog first before upgrading to version 6.1 or newer.

=> Proxy 6.1 and later stores spooled data differently from previous version. If the proxies in your environment have accumulated a substantial backlog, drain the backlog before upgrading to version 6.1 or later.